forge-gui/res/cardsfolder/upcoming/twice_upon_a_time_unlikely_meeting.txt

```diff
@@ -0,0 +1,15 @@
+Name:Twice Upon a Time
+ManaCost:4 U U
+Types:Sorcery
+A:SP$ AddTurn | NumTurns$ 1 | SubAbility$ DBChange | IsPresent$ Doctor.YouCtrl | PresentCompare$ GE2 | SpellDescription$ Take an extra turn after this one.
```

Comment:
You'll need to add the cast requirement to this SpellDescription for it to show in the Card Detail pane. Depending on how much you want it to match the card, you could do:
`SpellDescription$ Cast this spell only if you control two or more Doctors.,,,,,,Take an extra turn after this one.`